docs/index.md

```diff
@@ -1,27 +1,63 @@
+
+**VIAME Web** is the *specific* DIVE Web deployment at [viame.kitware.com](https://viame.kitware.com). It includes a web-based annotator with the capabilities to run VIAME workflows on user-provided data.  You may deploy the web system into your own lab or cloud environment.
+
+**Detection** - A single annotation.  A detection could be associated with a point in time within a track, or it could have no temporal association.
+
+**Features** - Bounding box, polygon, head/tail points or other visible elements of a detection.
+
+**Track** - A collection of detections spanned over multiple frames in a video or image sequence.  Tracks include a start and end time and can have gap periods in which no detections exist.
+
+**Types** - Every track (or detection, if tracks aren't applicable) has one or more types that should be used to annotate the primary characteristic you are interested in classifying.  Types are typically used to train a single or multi-class classifier.  A track (or detection) may have multiple types with confidence values associated.
 
-Current capabilities of DIVE include:
+**Frame** - A single image or point in time for a video or image sequence.
 
-* User import of frame images or video.
-* Playback of existing annotation data.
-* Manual creation of new annotations.
-* Automatic object detection and tracking of user-imported data.
-* Manual user refinement of automatically generated tracks
-* Export of generated annotations.
+**Key Frame** - Every manually drawn annotation is considered a keyframe, and all automated pipelines produce keyframes. Only keyframes can have attributes.  Key frame detections are differentiated from interpolated detections, which are the implicit bounding boxes you see when linear interpolation is enabled.
 
-## Definitions
+**Interpolation** - The implicit bounding boxes between keyframes in a track.
 
-* **DIVE** is the annotator and data management software system.  It is our name for the code and capabilities, including both web and desktop, that can be deployed and configured for a variety of needs.
-* **VIAME** (Video and Image Analytics for Marine Environments) is a suite of computer vision tools for object detection, tracking, rapid model generation, and many other types of analysis.  Get more info at [viametoolkit.org](https://www.viametoolkit.org/)
-* **VIAME Web** is the *specific* DIVE Web deployment at [viame.kitware.com](https://viame.kitware.com). It includes a web-based annotator with the capabilities to run VIAME workflows on user-provided data.  You may deploy the web system into your own labratory or cloud environment.
+**Attributes** - Attributes are free-form secondary annotations on both tracks and detections. For example, a `fish` type track may have an `is_adult` boolean attribute.
```

Comment:
does 'secondary annotations' invoke more visual items than data?

Reply:
switched to "secondary characteristic" to complement language in "Types"